composer.json

```diff
@@ -17,7 +17,8 @@
         "drupal/simplesamlphp_auth": "^3.0",
         "drupal/jsonapi_extras": "1.x-dev#1a57e11212224129ddb0229726dc7b852bcd836c",
         "drupal/maillog": "1.x-dev#14fd24cd89e299ad271608cd52706c60b753c26d",
-        "drupal/restui": "^1.15"
+        "drupal/restui": "^1.15",
+        "drupal/roleassign": "^1.0@alpha"
```

Comment:
Just a tip! Splitting up changes to `composer.json` and `composer.lock` into separate commits makes certain things easier, like rebasing for example. If you were to rebase onto the latest `develop` branch and it happened to have `composer.lock` changes, you could basically do away with your separate `composer.lock` commit and just run `composer install` again with your `composer.json` commit still intact. We used to do something similar back in D7 with make files, where we'd have one commit to update the file and a separate to run it. Similar concept with sass changes in one commit + compiled css changes in a separate one.